README.md

```diff
@@ -1,6 +1,5 @@
 # the FAIR extension
 
-[![npm version](https://badge.fury.io/js/npm.svg)](https://badge.fury.io/js/npm)
 [![npm version](https://img.shields.io/badge/license-Apache%202-blue)](https://opensource.org/licenses/Apache-2.0)
 [![fair-software.eu](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B-yellow)](https://fair-software.eu)
```

Comment:
It would be great if we can have the 5 stars badge

Reply:
Do you have an issue about this? I think this should be part of another PR?